pkg/validator/cluster.go

```diff
@@ -424,6 +425,28 @@ func validateRoleStorageClass(
 			// No storage section.
 			continue
 		}
+		// Validate storage size.
+		storageSize, err := resource.ParseQuantity(role.Storage.Size)
+		if err != nil {
+			valErrors = append(
+				valErrors,
+				fmt.Sprintf(
+					invalidStorageDef,
+					role.Name,
+				),
+			)
+			break
+		}
+		if storageSize.IsZero() {
```

Comment:
Can we check for a positive value here rather than just nonzero?

E.g. maybe `storageSize.Sign() == 1`